src/java/picard/util/BedToIntervalList.java

```diff
@@ -44,6 +44,12 @@
     @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output Picard Interval List")
     public File OUTPUT;
 
+    @Option(doc="If true, sort the output interval list before writing it.")
+    public boolean SORT = true;
+
+    @Option(doc="If true, unique the output interval list by merging overlapping regions, before writing it (implies sort=true).")
```

Comment:
Take a look at IntervalList.java:137 - the implementation of IntervalList.uniqued() is documented that it will sort the interval list prior to merging adjacent intervals.